pom.xml

```diff
@@ -167,12 +167,12 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-support</artifactId>
-        <version>2.22</version>
+        <version>${workflow-support-plugin.version}</version>
       </dependency>
       <dependency>
         <groupId>org.jenkins-ci.plugins.workflow</groupId>
         <artifactId>workflow-support</artifactId>
-        <version>2.22</version>
+        <version>${workflow-support-plugin.version}</version>
```

Comment:
Important to use properties to define versions of dependencies used with two classifiers, or you risk having them drift apart.